.github/workflows/fork-ci.yaml

```diff
@@ -60,6 +60,7 @@ jobs:
     - uses: docker/setup-qemu-action@v3
     - uses: docker/setup-buildx-action@v3
     - uses: docker/login-action@v3
+    - uses: chrisdickinson/setup-yq@v1.0.1
       with:
         registry: ${{ env.DOCKER_REGISTRY }}
         username: ${{ secrets.NVCR_USERNAME }}
```

Comment:
These parameters seem to have accidentally been moved out from the `docker/login-action` actions.